docs/quickstarts/insights-malware-install/insights-malware-install.yml

```diff
@@ -0,0 +1,206 @@
+          ```
+          test_scan: false
+          ```
+          b. Minimize scan time by setting any of the following options: 
+          ```
+          Filesystem_scan_only
+          ```
+          ```
+          Filesystem_scan_exclude
+          ```
+          ```
+          Filesystem_scan_since
+          ```
+          c. Re-run the client malware detection collector 
+          ```
+          sudo insights-client --collector malware-detection
+        ```
+      # optional - the task's Check your work module
+      review:
+        instructions: |-
+          - Did you complete the task successfully?
+        failedTaskHelp: This task isn't verified yet. Try the task again.
+      # optional - the task's success and failure messages
+      summary:
+        success: Shows a success message in the task header
+        failed: Shows a failed message in the task header
+
+
+    - title:  "Optional: Scan Processes"
+      description: |-
+        1. Edit 
+         ```
+         /etc/insights-client/malware-detection-config.yml
+         ```   
+          a. Set the test scan processes from false to true
+          ```
+          scan_processes: true
+          ```
+          b. Save changes
+        2. Run the malware detection collector again        
+          ```
+          sudo insights-client --collector malware-detection
+          ```
+      # optional - the task's Check your work module
+      review:
+        instructions: |-
+          - Did you complete the task successfully?
+        failedTaskHelp: This task isn't verified yet. Try the task again.
+      # optional - the task's success and failure messages
+      summary:
+        success: Shows a success message in the task header
+        failed: Shows a failed message in the task header
+
+    - title: Review results 
+      description: |-
+        Now you are ready to view your results.
+      introduction: |-
+        Welcome to the quick start for viewing your results. 
+        In this quick start, you will navigate through Insights to the Malware service to view the matched signatures. 
+  
+    - title: Review YARA Matches in Insights 
+      description: |-
+         1. Go to Red Hat Hybrid Cloud Console.
+         2. Navigate to **Insights for RHEL**.
+         3. Click **Security** in the left navigation panel. 
+         4. Select Malware in the dropdown.
+         5. You can click on **Systems** OR **Signatures** to see different views. 
+
+      # optional - the task's Check your work module
+      review:
+        instructions: |-
+          - Did you complete the task successfully?
+        failedTaskHelp: This task isn't verified yet. Try the task again.
+      # optional - the task's success and failure messages
+      summary:
+        success: Shows a success message in the task header
+        failed: Shows a failed message in the task header
+
+      conclusion: |-
+        <h4><b>Thank you for taking the time to explore this feature!</b></h4> 
```

Comment:
I can't see this line in the preview tool - not sure if there's a formatting issue?

Reply:
I chose an already published yaml file as a template. It had a conclusion, so I thought that was the standard and added it to Shayna's content. 

We chose just the installation for this quick start, but you could add something from the Malware overview that describes the Malware service:

> The Red Hat Insights for Red Hat Enterprise Linux malware detection service is a monitoring and assessment tool that scans RHEL systems for the presence of malware. The malware detection service incorporates YARA pattern-matching software and malware detection signatures. Signatures are provided in partnership with the IBM X-Force threat intelligence team working closely with the Red Hat threat intelligence team.
> 
> In the malware detection service UI, User Access-authorized administrators and viewers can
> 
> See the list of signatures against which their RHEL systems are scanned.
> See aggregate results for all RHEL systems with malware detection enabled in the Insights client.
> See results for individual systems.
> Know when a system shows evidence of the presence of malware.
> These features give security threat assessors and IT incident-response teams valuable information to prepare a response.